content/pipes.md

```diff
@@ -295,6 +295,22 @@ In this case, we want to bind the pipe at the method call level. In our current
 2. Pass the context-specific Joi schema in the class constructor of the pipe
 3. Bind the pipe to the method
 
+Joi schema example:
+
+```typescript
+const createCatSchema = Joi.object({
+  name: Joi.string().required(),
+  age: Joi.number().required(),
+  breed: Joi.string().required(),
+})
+
+export class CreateCatDto {
```

Comment:
Is there a reason to make this a class if we aren't making use of the metadata from using a class? Would this be better as an `interface`?

Reply:
Never tried it as an interface. If that's what you guys prefer, go for it.

Reply:
@jared-leddy only Kamil and the author of this PR has permissions to commit things.

Looks like the author is busy. Feel free to copy and paste his changes and open a another PR so we could merge it ASAP

Reply:
Sorry @micalevisk. Didn't realize it was under the wrong account. Changes made and updated.